src/agent/communicator/src/communicator.cpp

```diff
@@ -119,12 +121,12 @@ namespace communicator
                 const auto result = SendAuthenticationRequest();
                 if (result != boost::beast::http::status::ok)
                 {
-                    return std::chrono::milliseconds(A_SECOND_IN_MILLIS);
+                    return std::chrono::milliseconds(CONNECTION_RETRY_MSECS);
                 }
                 else
                 {
                     return std::chrono::milliseconds((GetTokenRemainingSecs() - TOKEN_PRE_EXPIRY_SECS) *
-                                                     A_SECOND_IN_MILLIS);
+                                                     1000); // NOLINT
```

Comment:
Instead of hardcoding the `1000` and adding the `NOLINT` keep the variable this PR is removing (`A_SECOND_IN_MILLIS`). There are a few occurrences of this in this file and others.

Reply:
Done